projects/pricefeed/src/price-oracle.ts

```diff
@@ -199,6 +199,8 @@ export class PriceOracle {
         case 'ubtc:usd':
         case 'ubtc:usd:30':
           return convertedPrice / 1000000;
+        case 'usdc:usd':
```

Comment:
Because in the backend, usdc will be treated as "u-usdc" as same as the other token, I think the price of usdc we post should be micro unit like ubtc.

Reply:
I see.
I will modify it

Reply:
done